### ISSUE TITLE
OpenGL implementation

### ISSUE THREAD

**Issue by weiyichang** [bwasti/AICamera#39]
Hi,
Thank for sharing this useful code so that we could run caffe2 on Android.
Recently, I notice that the OpenGL backend has been [released](https://github.com/caffe2/caffe2/tree/master/caffe2/contrib/opengl), and I also discover that the main difference is using GLPredictor. But the input and output of GLPredictor are different from the CPU version (Predictor), and the usage of GLImage/GLImageVector is lack of introduction. So I wanna know that would you plan to share this example with OpenGL backend, or could you give example of the transformation between TensorCPU and GLImage?
Thanks!!!

**Comment by rlegithub** [comment]
Yes, have the same question here. Do you have any plan for supporting latest Caffe2 which has new GL-backend implemented?

**Comment by jerryzh168** [comment]
OpenGL implementation is not stabilized yet, we may give a tutorial later.

**Comment by weiyichang** [comment]
@jerryzh168 Got it. Thanks a lot !!! 
Since the acceleration by GPU is great, we are looking forward to this implementation.
